packages/quantic/force-app/main/default/lwc/quanticUserActionsToggle/quanticUserActionsToggle.js

```diff
@@ -74,12 +76,16 @@ export default class QuanticUserActionsToggle extends LightningElement {
   initialize = (engine) => {
     this.headless = getHeadlessBundle(this.engineId);
 
+    if (this.excludedCustomActions?.length) {
+      this._excludedCustomActions = [...this.excludedCustomActions];
+    } else {
+      this._excludedCustomActions = [];
+    }
```

Comment:
I don't understand this fix... if it already had a length, put it in another array to use it?

If it didn't have a length, put an empty array?

Reply:
yes indeed, 
when the component receives the array as property from a parent component salesforce transforms this object to a proxy since it's not of a primitive type, and when passing this proxy object to headless the error mentioned in the PR descriptions is logged.

To avoid this, I'm making sure with this manipulation that I'm passing a real array to Headless instead of proxy array/

Reply:
Wouldn't it make more sense in this case to have the option be a primitive string?
It would also mean it could be set as a HTML property easily?

And we parse the string where each value is separated by a `,` as well?

Reply:
generally speaking I'm not  big fan of passing an array as a string but in this case with SF transforming evrything to a proxy it makes a lot of sense x)

but this would be considered a breaking change at this point